distribution/changelog.txt

```diff
@@ -6,6 +6,7 @@
 - Feature: [#5576] Add a persistent 'display real names of guests' setting.
 - Feature: OpenRCT2 now starts up on the display it was last shown on.
 - Improved: Mouse can now be dragged to select scenery when saving track designs
+- Feature: [#5504] Group vehicles into ride groups
```

Comment:
Changelog is ordered by: `type, PR/Issue (with non-numbered ones at the end), alphabetical`